scripts/pi-hole/php/add.php

```diff
@@ -51,14 +51,22 @@
 switch($list) {
 	case "white":
 		$domains = array_map('strtolower', $domains);
+		remove_from_table($db, "domainlist", $domains, $returnnum=false, $type=1);
```

Comment:
Do you really want to specify the type here? Imagine a domain was added as regex previously. In this case, your removal would not get it. When you don't specify the type, it will always be removed.

The same goes for the `black` case below.

Reply:
If no type is specified, deletion does not work. The request returns information, but the item is still on the list.

Tested for:

http://pi.hole:8085/admin/api.php?list=black&add=example.com&auth=api_key_here

http://pi.hole:8085/admin/api.php?list=white&add=example.com&auth=api_key_here

http://pi.hole:8085/admin/api.php?list=none&add=example.com&auth=api_key_here

The modification should work for the plugin:
https://github.com/badsgahhl/pihole-chrome-extension/issues/7